packages/default-storage/android/build.gradle

```diff
@@ -143,43 +94,36 @@ android {
 repositories {
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "${resolveModulePath("react-native")}/android"
+        url "${project.ext.resolveModulePath("react-native")}/android"
     }
     google()
     mavenCentral()
 }
 
 dependencies {
-
     if (useNextStorage) {
-        def room_version = getVersionOrDefault('AsyncStorage_next_roomVersion', '2.4.3')
-        def coroutines_version = "1.6.4"
-        def coroutinesTest_version = "1.6.4"
-        // if we don't provide explicit dependency on reflection, kotlin plugin
-        // would add one automatically, probably a version that is not compatible with
-        // used kotlin
-        def kotlinReflect_version = project.ext.asyncStorageKtVersion
+        def room_version = project.ext.AsyncStorageConfig.roomVersion
+        def coroutines_version = "1.7.3"
         def junit_version = "4.13.2"
-        def robolectric_version = "4.7.3"
+        def robolectric_version = "4.11.1"
         def truth_version = "1.1.3"
         def androidxtest_version = "1.4.0"
         def androidtest_junit_version = "1.1.3"
```

Comment:
I think we should move these into `config.gradle` as well. That way we've got all version numbers in a single place. Alternatively, if we set it up similarly to RNTA's `dependencies.gradle`, we can get Dependabot to automatically bump the version numbers as well.

Reply:
Nice, moved it to `config.gradle`